pygmt/helpers/tempfile.py

```diff
@@ -12,17 +12,17 @@
 from packaging.version import Version
 
 
-def unique_name():
+def unique_name() -> str:
     """
     Generate a unique name.
 
-    Useful for generating unique names for figures (otherwise GMT will plot
-    everything on the same figure instead of creating a new one).
+    Useful for generating unique names for figures (otherwise GMT will plot everything
+    on the same figure instead of creating a new one).
```

Comment:
```suggestion
    Useful for generating unique names for figures. Otherwise GMT will plot everything
    on the same figure instead of creating a new one.
```